### PR TITLE
♻️ Unifying frontend retrieve call

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

legacy and dy-sidecar dynamic services now use the same endpoint to retrieve inputs.

Previously `legacy` services were directly calling the service's endpoint to retrieve the state., Now it is called via `director-v2`, using the same endpoint as the new style services.

### CODE REVIEW

**Review by sanderegg — APPROVED** [review]
lgtm

**Review by pcrespov — COMMENTED** [review]
👍

File: services/web/client/source/class/osparc/utils/Utils.js [hunk 1]
```diff
@@ -128,12 +128,7 @@ qx.Class.define("osparc.utils.Utils", {
...
     },
 
-    computeServiceRetrieveUrl: function(srvUrl) {
-      const urlRetrieve = srvUrl + "/retrieve";
-      return urlRetrieve.replace("//retrieve", "/retrieve");
-    },
-
-    computeServiceV2RetrieveUrl: function(studyId, nodeId) {
+    computeServiceRetrieveUrl: function(studyId, nodeId) {
       const urlBase = window.location.protocol + "//" + window.location.host + "/v0";
```
Comment (odeimaiz) [inline services/web/client/source/class/osparc/utils/Utils.js:132]: I'm not 100% sure this works for every legacy dynamic services we support. Please, check with the usual suspects: 3D viewer, 2D plot

**Comment by pcrespov** [comment]
@GitHK do not forget to set all attributes of your PRs (I already did it for you this time :-) )

File: services/web/client/source/class/osparc/utils/Utils.js (hunk 1 above)
Comment (GitHK) [reply services/web/client/source/class/osparc/utils/Utils.js:132]: So 2D plot seems ok with the new API.
**3D viewer** is not happy, more investigation is required. Apparently it returns a 404 Not found when queried by the backend (log appears in the webserver).

**Comment by GitHK** [comment]
closing this, not a way to go

**Pull request closed without merging**